unit_tests/test_dockeropts.py

```diff
@@ -0,0 +1,38 @@
+from lib.dockeropts import DockerOpts
+
+
+class TestDockerOpts:
+
+    def test_init(self):
+        d = DockerOpts()
+        assert isinstance(d.data, dict)
+
+    def test_add(self):
+        d = DockerOpts()
+        d.add('foo', 'bar')
+        assert 'foo' in d.data
+        assert d.data['foo'] == ['bar']
+
+    def test_add_multi(self):
+        d = DockerOpts()
+        d.add('foo', 'bar, baz')
+        assert 'foo' in d.data
+        assert d.data['foo'] == ['bar', 'baz']
+
+    def test_to_s(self):
+        d = DockerOpts()
+        d.add('foo', 'bar, baz')
+        assert d.to_s() == "--foo=bar --foo=baz"
+
+    def test_remove_single(self):
+        d = DockerOpts()
+        d.add('foo', 'bar, baz')
+        d.remove('foo', 'baz')
+        assert 'baz' not in d.data['foo']
+        assert 'bar' in d.data['foo']
+
+    def test_data_persistence(self):
+        x = DockerOpts()
+        x.add('juju', 'is amazing')
```

Comment:
Correct. Its checking data persistence across objects. They're backed by SQLite and a new instance "reloads" the data from the sqlite data store. If we ever encounter a situation where 2 modules are modifying the options for the docker daemon, they are not likely to use the same object, so i thought a test was prudent :)